core/modules/system/config/system.core.json

```
@@ -91,6 +91,8 @@
     "user_mail_status_canceled_notify": 0,
     "user_register": "admin_only",
     "user_signatures": 0,
+    "user_page_tabs": 0,
+    "user_page_suppress_layout": 1,
```

Can we please explicitly add the word `login` to these settings?:
```suggestion
    "user_login_page_tabs": 0,
    "user_login_page_suppress_layout": 1,
```